### ISSUE TITLE
Can't install postmark provider version 0.2.2 with Terraform v1.3.6

### ISSUE THREAD

**Issue by nhybytes** [shebang-labs/terraform-provider-postmark#1]
# Issue Description
When trying to install Postmark provider in Terraform Cloud with terraform version `v.1.3.6`, Terraform cloud ended errors

```
╷
│ Error: Failed to install provider
│ 
│ Error while installing shebang-labs/postmark v0.2.2: checksum list has
│ unexpected SHA-256 hash
│ 205a9afffbb1c5a7fe79dd27677faf67c2fa0b655e07a4e4cf0fe5d1c67e650c (expected
│ 1138951e4db70a3e7c52828c14c120471af5d8d07ddfc7e0034abd57f4f7b422)
```

# What have expected
Terraform cloud should install postmark provider successfully

# Steps to produce

Here are the steps have been done

### Create postmark configuration

```
terraform {
  required_providers {
    postmark = {
      source  = "shebang-labs/postmark"
      version = "0.2.2"
    }
  }
}

# POSTMARK provider configuration
provider "postmark" {
  account_token = var.postmark_account_token
}
```

### Push to Terraform cloud

Then commit to Terraform Cloud

**Comment by DennisVonDerBey** [comment]
Installing version `0.2.1` works. I guess `0.2.2` hasn't been published fully.

**Comment by nhybytes** [comment]
> Installing version `0.2.1` works. I guess `0.2.2` hasn't been published fully.

Version `0.2.1` works perfectly

**Comment by MuhammeedAlaa** [comment]
Hello @nhybytes sorry for the late reply, this issue is fixed in version `0.2.3` please install it and give us feedback if you have any 😄
